tests/unit/test_install.py

```diff
@@ -834,3 +839,171 @@ def test_repair_run_result_state(ws, caplog, mock_installation_with_jobs, any_pr
 
     workspace_installation.repair_run("assessment")
     assert "Please try after sometime" in caplog.text
+
+
+@pytest.mark.parametrize(
+    "state,expected",
+    [
+        (
+            RunState(
+                result_state=None,
+                life_cycle_state=RunLifeCycleState.RUNNING,
+            ),
+            "RUNNING",
+        ),
+        (
+            RunState(
+                result_state=RunResultState.SUCCESS,
+                life_cycle_state=RunLifeCycleState.TERMINATED,
+            ),
+            "SUCCESS",
+        ),
+        (
+            RunState(
+                result_state=RunResultState.FAILED,
+                life_cycle_state=RunLifeCycleState.TERMINATED,
+            ),
+            "FAILED",
+        ),
+        (
+            RunState(
+                result_state=None,
+                life_cycle_state=None,
+            ),
+            "UNKNOWN",
+        ),
+    ],
+)
+def test_latest_job_status_states(ws, mock_installation_with_jobs, any_prompt, state, expected):
+    base = [
+        BaseRun(
```

Comment:
Use the patter introduced in test_clusters.py to externalize Base Run information

Reply:
No now, as discussed.